deploy/complete/helm-chart/mushop/charts/carts/templates/carts-deployment.yaml

```diff
@@ -7,13 +7,18 @@ spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
+      app: {{ .Chart.Name }}
+      version: {{ .Values.image.tag | quote }}
       app.kubernetes.io/name: {{ include "carts.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
   template:
     metadata:
       labels:
+        app: {{ .Chart.Name }}
+        version: {{ .Values.image.tag | quote }}
         app.kubernetes.io/name: {{ include "carts.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        mockmode: {{ .Values.global.mock.service | quote }}
```

Comment:
any reason mockmode needs to be in the labels?

Reply:
Just to trigger a new pod when the connected services actually get connections, even if the config/secrets are unchanged

Reply:
But could just as easily be an env. Opinions?